src/tex/ms.tex

```diff
@@ -120,7 +120,8 @@ \section{Introduction}
 \section{Prior}
 \label{sec:prior}
 
-The most common prior probability that is used for a parameter is uniform between two values.
+% ARJN Application of a half-closed distribution may be as common? e.g. [0, inf] for a parameter that is known to be non-negative. Should we mention this in table 1?
```

Comment:
It's a half closed distribution possible in a Bayesian sampler? Would this not lead to an 0 probability given that $p(U) = \frac{1}{b-a}$, where one of $a$ or $b$ would be infinity?

Reply:
With emcee and pymc (v3/4) I know that half-closed distributions are possible for priors. In the case of emcee you would do:

```
def log_prior(theta):
    # parameter specified in [0, inf]
    if theta < 0:
        return -inf
    return 0
```

i.e. probability is not "scaled" correctly, in that the log-prior has no numerical effect if the parameter is in a possible region. However, if the parameter is in an infeasible region the log-prior makes the probability -np.inf (impossible).

Reply:
Oh yeah, that is a bit daft of me cause I wrote code that does exactly that earlier...It is probably worth a passing mention then.